paymentsheet/src/main/java/com/stripe/android/paymentsheet/FormFragment.kt

```diff
@@ -0,0 +1,42 @@
+package com.stripe.android.paymentsheet
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import com.stripe.android.paymentsheet.forms.SofortForm
+import com.stripe.android.paymentsheet.forms.SofortFormViewModel
+
+class FormFragment : Fragment() {
+    val sofortFormViewModel: SofortFormViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View = ComposeView(inflater.context).apply {
+        layoutParams = ViewGroup.LayoutParams(
+            ViewGroup.LayoutParams.MATCH_PARENT,
+            ViewGroup.LayoutParams.MATCH_PARENT
+        )
+
+        setContent {
+            StripeTheme {
+                Column(Modifier.fillMaxSize()) {
+                    val param by sofortFormViewModel.params.observeAsState(null)
+                    SofortForm(sofortFormViewModel)
+                    Log.e("APP", "Params: $param")
```

Comment:
Do we want to keep this log? Maybe use info or debug level? Also a better tag?

Reply:
I will move this into the example activity instead of this Fragment.